src/eko/quantities/heavy_quarks.py

```diff
@@ -65,19 +65,6 @@ def t(self, value: T):
 MatchingScales = HeavyQuarks[MatchingScale]
 
 
-def scales_from_ratios(
-    ratios: MatchingRatios, masses: HeavyQuarkMasses
-) -> MatchingScales:
-    """Convert ratios to squared scales.
-
-    .. todo::
-
-        make this a method
-
-    """
-    return MatchingScales(*(np.power(ratios, 2.0) * np.power(masses, 2.0)).tolist())
```

Comment:
@AleCandido you should remove that ambiguity here https://github.com/NNPDF/eko/blob/ec083e7ed167554107defd3723c50008046e596a/src/eko/quantities/heavy_quarks.py#L61 which was make this function failing - and since it is not actually used anywhere I decided to to the simplest thing and remove it all together

Reply:
Ah, I see. But that ambiguity I need for parsing, since we might have pole masses or MSbar ones.

The alternative is to make them always `QuarkMassRef`, slightly increasing the runcards complexity, but with the benefit of having a single type around (the reference for pole masses can be `nan`)

Reply:
Ok, so we can drop the `Union`. If we were able to completely avoid `Union`s (apart from `Optional`, that's a different story) my life would be much easier :)